build/Scripts/copyOutput.cmd

```diff
@@ -0,0 +1,5 @@
+echo Copy the Modern Vsixes manifest into VsixV3
+robocopy %BinariesDirectory% %BinariesDirectory%VsixV3 Microsoft.VisualStudio.Editors.vsman Microsoft.VisualStudio.ProjectSystem.Managed.vsman /njh /njs /np /xx
+
+echo Copy the Nugets to CoreXT Share
+robocopy %BinariesDirectory%NuGetPackages \\cpvsbuild\drops\dd\nuget /njh /njs /np /xx 
```

Comment:
Yes, it does. I had it tested with test build definition. The results are [here](https://devdiv.visualstudio.com/DevDiv/_build/index?buildId=616735&_a=summary)